internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go

```diff
@@ -1111,3 +1135,7 @@ func readBackSensitiveValue(input []interface{}, propertyName string, filters ma
 	}
 	return ""
 }
+
+func expandDataFactoryIntegrationRuntimeElasticPool(input string) string {
```

Comment:
Can we rename this function to `formatDataFactoryIntegrationRuntimeElasticPool`?